src/plugins/saved_objects_management/server/routes/bulk_delete.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { IRouter } from '@kbn/core/server';
+
+export const registerBulkDeleteRoute = (router: IRouter) => {
+  router.post(
+    {
+      path: '/api/kibana/management/saved_objects/_bulk_delete',
+      validate: {
+        body: schema.arrayOf(
+          schema.object({
+            type: schema.string(),
+            id: schema.string(),
+          })
+        ),
+      },
+    },
+    router.handleLegacyErrors(async (context, req, res) => {
+      const { getClient } = (await context.core).savedObjects;
+
+      const objects = req.body;
+      const client = getClient();
```

Comment:
We can if we want, as we do that for other SOM routes already:

https://github.com/elastic/kibana/blob/8c19c36b36e23aab26cefea9ed85df18c71d882d/src/plugins/saved_objects_management/server/routes/find.ts#L60-L64

Reply:
> But doing that for the table view doesn't seem that easy from the UX point.

Agreed, I've been considering different options since adding the bulk_delete API but non of them are simple. I don't think we need to jump in supporting bulk_delete for hidden types just yet. It's not a trivial task.